src/lib/containers/row_renderers/Dataset.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import * as Utils from './utils'
+
+const DATASET_SCHEMA = {
+    id: 0,
```

Comment:
Please change all of these to discover the column indices from the query result headers (Table/View results may contain many unrelated columns in the future).